Specify `/tmp/balena|resin` directories as necessary (#2391)
balena-os/balena-supervisor: docs/update-locking.md

### Location of the lockfile

> __Note:__ Please do not try to delete `/tmp/balena` or legacy `/tmp/resin` directories, as you will get an error complaining of a busy resource.

Reviewer: Github Markdown also supports this format, but I'm not sure if it is supported by our docs backend

```suggestion
> [!NOTE]  
> Trying to remove`/tmp/balena` or legacy `/tmp/resin` directories will result in an `resource busy` error
```

Author: Just scanning through the [local mode](https://github.com/balena-io/docs/edit/master/pages/learn/develop/local-mode.md) docs, it looks like `__Note:__` is the preferred syntax
